File: programmers/lv0/코딩기초트레이닝/연산/문자열섞기/Solution.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string str1, string str2) {
    vector<char> v1(str1.begin(), str1.end());
    vector<char> v2(str2.begin(), str2.end());

    string answer;
    for (int i = 0; i < v1.size(); ++i) {
        answer += v1[i];
        answer += v2[i];
    }

    return answer;
}

File: programmers/lv0/코딩기초트레이닝/연산/문자열섞기/Solution.java
package lv0.코딩기초트레이닝.연산.문자열섞기;

class Solution {
    public String solution(String str1, String str2) {
        String answer = "";
        for (int i = 0; i < str1.length(); i++) {
            answer += str1.charAt(i);
            answer += str2.charAt(i);
        }
        return answer;
    }
}